### ISSUE TITLE
Pd resources - Transgenic 

### ISSUE THREAD

**Issue by nrandel** [platynereis/resources#3]
I found the link to the page (multiple tabs), which was created by Leslie from your lab.

https://docs.google.com/spreadsheets/d/1fI1SS-_eih-OkBXlIQAcxVJ4O0tH3jI5EyH-RyQ8KG0/edit#gid=0

I think this needs to be implemented here and maybe can be condensed.

**Comment by platynereis** [comment]
will look into it. Should we condense it into one big table? We could have a column about the status (worked/didn't work/planned), but the other fields don't overlap that much. Also, the information that is entered is incomplete in most cases >.>

**Comment by nrandel** [comment]
Yes, I would prefer your suggestion to put everything in one table
